moredis/dbs.go

```diff
@@ -30,6 +39,34 @@ func SetupDbs(mongoURL, redisURL string) (*mgo.Database, redis.Conn, error) {
 	return mongoDB, redisConn, nil
 }
 
+// resolveRedis takes in a redis address and checks for a sentinel:// prefix to resolve. If one is present, it uses
+// sentinel to resolve the master and returns that. Otherwise, it returns the original address.
+func resolveRedis(address string) (string, error) {
+	if !strings.HasPrefix(address, "sentinel://") {
+		return address, nil
+	}
+
+	match := regexp.MustCompile(`sentinel://([^/]+)/(.*)`)
+	addressParts := match.FindAllStringSubmatch(address, -1)
+
+	if len(addressParts) < 1 || len(addressParts[0]) < 3 {
```

Comment:
What is the structure of `addressParts` here? I read the docs for `FindAllStringSubmatch` but in typical godoc fashion they said almost nothing.